kuma/static/js/experiment-interactive-editor.js

```diff
@@ -0,0 +1,14 @@
+(function() {
+    'use strict';
+
+    var cop = new Mozilla.TrafficCop({
+        id: 'experiment-interactive-editor',
+        cookieExpires: 24 * 365, // 1 year
+        variations: {
+            'v=a': 50, // control
+            'v=b': 50 // interactive editor
+        }
+    });
+
+    cop.init();
+})(window.Mozilla);
```

Comment:
Should ``Mozilla`` be an argument of this anon function? It doesn't make sense to me that ``window.Mozilla`` is passed in otherwise, but I'm a lightweight JS dev, and this version works.

Reply:
I actually do not even think we need to pass this in at all, I will test. As `Mozilla` and all objects attached to it are exposed to the global `window` object, it should just be available. I believe this was from a version I copy and pasted, and was also wondering whether it was really required.